WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DumpDeveloperDatabase < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Create developer database dump every 3 days.
+    last_developer_db_dump = Timestamp.find_or_create_by(name: 'last_developer_db_dump')
+    if last_developer_db_dump.date.nil? || last_developer_db_dump.date < 3.days.ago
```

Comment:
Good point with the name, I changed to `not_after?`.
`not_since?` sounds good, but I have a feeling like it implies that the timestamp continuously changes (like if its only one usage were jobs), whereas `not_after?` is more neutral and fits here better in my opinion =)